src/ui/public/courier/saved_object/saved_object.js

```diff
@@ -282,7 +282,7 @@ export default function SavedObjectFactory(esAdmin, kbnIndex, Promise, Private,
           if (_.get(err, 'origError.status') === 409) {
             const confirmMessage = `Are you sure you want to overwrite ${this.title}?`;
 
-            return safeConfirm(confirmMessage)
+            return confirmModalPromise(confirmMessage, { confirmButtonText: 'Overwrite' })
```

Comment:
Can that be a separate issue?  Feels like a chore/clean up and not necessary for this rename button text PR.